src/index.tsx

```diff
@@ -34,88 +47,100 @@ export const Zage = ({
   setShowZage,
 }: ZageProps) => {
   const zageApp = publicKey.startsWith('sandbox_') ? SB_APP_URL : PROD_APP_URL;
+  const [oAuth, setOAuth] = useState<boolean>(false);
+  const jsResponse =
+    baseJs +
+    `
+    const meta = document.createElement('meta'); meta.setAttribute('content', 'width=device-width user-scalable=0'); meta.setAttribute('name', 'viewport'); document.getElementsByTagName('head')[0].appendChild(meta);
+    openPayment('${paymentToken}', '${publicKey}')
+    true
+  `;
 
-  const [jsResponse, setJsResponse] = useState<string>('');
-
-  const getJsRes = async (zagePaymentToken: string) => {
-    try {
-      const req = new XMLHttpRequest();
-      req.onreadystatechange = () => {
-        const appendedJs =
-          req.responseText +
-          `
-          openPayment('${zagePaymentToken}', '${publicKey}')
-          true
-        `;
-        setJsResponse(appendedJs);
-      };
-      req.open('GET', 'https://api.zage.dev/v0/v0-rn.js', true);
-      req.send(null);
-    } catch (error) {
-      console.error(error);
-    }
-  };
-
-  useEffect(() => {
-    if (jsResponse === '' && paymentToken) {
-      getJsRes(paymentToken);
-    }
-  }, [paymentToken, jsResponse]);
-
+  if (!showZage) {
+    return <></>;
```

Comment:
```suggestion
    return null;
```
Purely a matter of style so feel free to ignore